src/main/webapp/WEB-INF/tags/navBarBrand.tag

```diff
@@ -0,0 +1,5 @@
+<%@ tag description="TEAMMATES logo brand"%>
+
+<a class="navbar-brand" href="/">
+    <img height="100%" src="../images/teammateslogo.jpg" alt="TEAMMATES[Logo] - Online Peer Feedback/Evaluation System for Student Team Projects">
```

Comment:
Does the source need to be `../images/teammateslogo.jpg`? Will simply `images/teammateslogo.jpg` work?